interface/usergroup/addrbook_edit.php

```diff
@@ -87,7 +87,7 @@ function invalue($name) {
   // 2 = Person Centric
   // 3 = Company Centric
   $sql = sqlStatement("SELECT option_id, option_value FROM list_options WHERE " .
-   "list_id = 'abook_type'");
+   "list_id = 'abook_type' AND activity = 1");
```

Comment:
Oh, nevermind this is list options type selector, not user active.   Is "activity" really the field name, does not seem correct.